.gitignore

```diff
@@ -3,9 +3,13 @@
 .DS_Store
 .ipynb_checkpoints
 .sass-cache
+.jekyll-cache/
 __pycache__
 _site
 .Rproj.user
 .Rhistory
 .RData
-.jekyll-cache
+.bundle/
+.vendor/
+.docker-vendor/
+Gemfile.lock
```

Comment:
Why would you ignore Gemfile.lock? `bundler` tells you to commit it, doesn't it, to enable reproducable builds?

Reply:
I agree, I'd usually commit `Gemfile.lock` as well as `Gemfile`, but I'll wait for @fmichonneau's comment as this seems to be how it's done in [carpentries/carpentries-theme](https://github.com/carpentries/carpentries-theme).

Reply:
We don't commit the lock file because we always want to use the latest version of the `github-pages`.